CHANGELOG.md

```diff
@@ -55,6 +55,7 @@ There are breaking changes in this release. Please see the *Features* section be
 - [#3672](https://github.com/influxdb/influxdb/pull/3672): Reduce in-memory index by 20%-30%
 - [#3673](https://github.com/influxdb/influxdb/pull/3673): Improve query performance by removing unnecessary tagset sorting.
 - [#3676](https://github.com/influxdb/influxdb/pull/3676): Improve query performance by memomizing mapper output keys.
+- [#3687](https://github.com/influxdb/influxdb/issues/3687): panic: runtime error: makeslice: len out of range
```

Comment:
If this is only in HH code, it might help to mention that it addresses a panic in HH.